styles/components/hero.scss

```diff
@@ -0,0 +1,31 @@
+.flex {
+  min-width: 20rem;
+  padding: 5rem;
```

Comment:
This padding is making it seem like the contents are centered.

If you remove the padding and set `min-height: 40vh` you'll see that this is the element that should be display:flex and then the parent flex properties can be used to center child content.

As a nice + this will make it so that you can remove almost all of the styling to the `&__text` for this element!